reactor-error-prone/src/main/java/com/github/lhotari/reactor/errorprone/ReactorInnerPublisherIgnored.java

```diff
@@ -61,7 +63,9 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
                 0,
                 state.getTypes());
 
-        if (typeArg == null || !(ASTHelpers.isSubtype(fluxType, typeArg, state) || ASTHelpers.isSubtype(monoType, typeArg, state))) {
+        if (typeArg == null
+                || ASTHelpers.isSubtype(objectType, typeArg, state)
+                || !(ASTHelpers.isSubtype(fluxType, typeArg, state) || ASTHelpers.isSubtype(monoType, typeArg, state))) {
```

Comment:
Thought: `fluxType` and `monoType` checks could be collapsed into a single `publiserType` check. WDYT?

Reply:
I think it's fine for now to keep them separate so that the rules target only Project Reactor types.